src/bikeracks/votes.py

```diff
@@ -104,7 +104,6 @@ def submit_vote():
         resp = get_vote_data(rack_id, user_id)
         return jsonify(resp)
     except Exception as e:
-        with open('/tmp/bikerack.log', 'a') as f:
-            f.writeline(str(e))
+        raise(e)
```

Comment:
This does nothing, actually. You catch the exception and just immediately raise it again. You might as well just not catch it at all.

Reply:
Okay, I changed it. Going to open a separate pull request for handling the submitVote errors on the client side. Also I moved a few things into their own directories.